cves/kernel/CVE-2013-0309.yml

```diff
@@ -55,7 +55,9 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
```

Comment:
Large transparent pages and Transparent Huge Pages aren't the same 😂, look up THP's and briefly elaborate on that + elaborate on what kind of system crash is caused by the attacker.

Reply:
Gotcha, probably should've looked that up instead of taking it so literally. Thanks.